terragrunt/modules/docs-rs/builder.tf

```diff
@@ -1,5 +1,50 @@
-// The instance profile the builder will assume when communicating with
-// other AWS services.
+// The autoscaling group for the builder
+
+resource "aws_autoscaling_group" "builder" {
+  name                = "docs-rs-builder"
+  vpc_zone_identifier = var.cluster_config.subnet_ids
+  max_size            = var.max_num_builder_instances
+  min_size            = var.min_num_builder_instances
+  # Let the instances get warm
+  default_instance_warmup = 60
+
+  launch_template {
+    id      = aws_launch_template.builder.id
+    version = "$Latest"
+  }
+}
+
+resource "aws_launch_template" "builder" {
+  name_prefix   = "builder"
+  image_id      = data.aws_ami.builder.id
+  instance_type = "t2.large"
+
+  network_interfaces {
+    associate_public_ip_address = true
+    security_groups             = [aws_security_group.builder.id]
+  }
+
+  iam_instance_profile {
+    arn = aws_iam_instance_profile.builder.arn
+  }
+
+  block_device_mappings {
+    device_name = "/dev/sda1"
+
+    ebs {
+      volume_size           = 64
```

Comment:
Seeing the current filesystem usage (~100 GB) I would prefer something at least double this size. 

( of course the current usage also includes the database & some web cache)

Reply:
It would be nice if we didn't need so much storage. I believe a large part of that storage is only needed during a single crate's build and afterwards can be deleted, no? Would it be possible to add some clean up to the builder process so that the filesystem usage doesn't grow so large?

Reply:
Hm.. we _are_ cleaning up after the build. 

Plus the cleanup tasks for docker images, which are in cron right now. 
( btw, cc @Nemo157 @jyn514 , these cronjobs would need to be configured in our ansible images too, right? )

Only looking at the above I could totally imagine to just try with the current definition above, let the builder build, and watch how much space is used. ( assuming the big docker image is configured?)

But, we're also planning on adding some build artifact caching: https://github.com/rust-lang/docs.rs/issues/1757 
( of course we could increase storage only then, when that feature is finished)

Reply:
Yep, we just have a daily cronjob running `docker container prune --force && docker image prune --force` (and cargo-sweep which shouldn't be necessary if we rebuild the image for a new version?).

Reply:
Where is the cronjob currently configured? I can add this to the Ansible configuration (though I wouldn't block merging this).

Reply:
It's in

```
/etc/systemd/system/prune-disk-space.service
/etc/systemd/system/prune-disk-space.timer
```